edgelet/Cross.toml

```diff
@@ -5,8 +5,14 @@ passthrough = [
     "IOTEDGE_HOMEDIR",
 ]
 
+[target.x86_64-unknown-linux-gnu]
+image = "edgebuilds.azurecr.io/debian-build:9.5-1"
```

Comment:
Didn't we want to host this in Docker Hub or in some public registry?

Reply:
Sure.